content/docs/ops/maintenance-list.md

```diff
@@ -12,24 +12,16 @@ personally, you are responsible for ensuring that an appropriate person does it.
 If you haven't already reached out on Slack, reach out during standup to get
 visibility with the people who might best help.
 
-# Is it your first day of support?
+# Are you new to platform support?
 
-- Update the [`#cg-platform`](https://gsa-tts.slack.com/messages/cg-platform/)
-topic to include your name as the support contact.
-- Update the support schedule by moving yourself to the end of the list in the
-[slackbot auto response](https://gsa-tts.slack.com/customize/slackbot) for
-`atlas support schedule`.
 - Join/unmute [`#cg-support`](https://gsa-tts.slack.com/messages/cg-support/)
-and [`#cg-platform-news`](https://gsa-tts.slack.com/messages/cg-platform-news/)
-- Meet with the previous support person and take responsibility for any open
-support items they are still working on. There is a standing support sync meeting
-(Tuesday 4:30-5 pm ET) that you should join if you are rolling on or off support
-that week.
 
-## At least once during your week of support
 
+## Weekly support tasks
+
+- Update the #cg-support topic to include your name as the support contact.
```

Comment:
I think we should add this back to the top heading, and revert back to "is this your first day of support"

Reply:
I'm fine reverting back to the original header name (Is this your first day of support).